benchmarks/username.txt

```diff
@@ -0,0 +1 @@
+farizrahman4u
```

Comment:
what are these usernames for? maybe rename the file to be more obvious? also, instead of using your personal username, maybe we should create an activeloop one